about/contributing_datasets.md

```diff
@@ -0,0 +1,288 @@
+```
+
+### Step 2b: Check In the Preprocessing Script.
+
+Check in the script and the resulting CSV to [https://github.com/datacommonsorg/data](https://github.com/datacommonsorg/data) under the appropriate [`scripts/<provenance>/<dataset>` subdirectory](https://github.com/datacommonsorg/data/tree/master/scripts/covid_tracking_project/historic_state_data).
+
+## Step 3: Specify the Template MCF.
+
+Template MCF is essentially a mapping file that instructs Data Commons on how to convert each row of a CSV into MCF. Each entity and `StatisticalVariable` will have a template. For additional information, read [Template MCF](https://github.com/datacommonsorg/data/blob/master/docs/mcf_format.md#template-mcf).
+
+The following is the mapping for the first two `StatisticalVariable`s.
+
+```
+Node: E:COVIDTracking_States->E0
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:CumulativeCount_MedicalTest_COVID_19
+measurementMethod: dcs:CovidTrackingProject
+observationAbout: C:COVIDTracking_States->GeoId
+observationDate: C:COVIDTracking_States->Dates
+value: C:COVIDTracking_States->CumulativeCount_MedicalTest_COVID_19
+
+Node: E:COVIDTracking_States->E1
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:CumulativeCount_MedicalTest_COVID_19_Positive
+measurementMethod: dcs:CovidTrackingProject
+observationAbout: C:COVIDTracking_States->GeoId
+observationDate: C:COVIDTracking_States->Dates
+value: C:COVIDTracking_States->CumulativeCount_MedicalTest_COVID_19_Positive
+
+... etc ...
+```
+
+The mapping for the remaining variables is essentially identical and can be generated by a script.
+
+**Note:** This example adds the property `measurementMethod: dcs:CovidTrackingProject` to distinguish the same CumulativeCount_MedicalTest_COVID_19 observation for a state and date from multiple sources (such as the New York Times, the COVID Tracking Project, or Harvard).
+
+```
+# Automate Template MCF generation since there are many Statistical Variables.
+TEMPLATE_MCF_TEMPLATE = """
+Node: E:COVIDTracking_States->E{index}
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:{stat_var}
+measurementMethod: dcs:CovidTrackingProject
+observationAbout: C:COVIDTracking_States->GeoId
+observationDate: C:COVIDTracking_States->Dates
+value: C:COVIDTracking_States->{stat_var}
+"""
+```
+
+Check in the Template MCF file together with the cleaned CSV and its preprocessing script to [https://github.com/datacommonsorg/data](https://github.com/datacommonsorg/data) under the appropriate [`scripts/<provenance>/<dataset>` subdirectory](https://github.com/datacommonsorg/data/tree/master/scripts/covid_tracking_project/historic_state_data). The Template MCF generation code may also be included (as in this example).
+
+## Conclusion
+
+After producing all three required components (`StatisticalVariable` schema MCF, cleaned CSV, and Template MCF), go ahead and submit a PR to [https://github.com/datacommonsorg/data](https://github.com/datacommonsorg/data).
```

Comment:
Should this link to the /pulls page?

Reply:
That's a great idea! Yes, I will incorporate that.